src/categorical_algebra/CSets.jl

```diff
@@ -659,8 +677,10 @@ be mutated even when the assignment fails.
 
     # Check attributes first to fail as quickly as possible.
     X, Y = state.dom, state.codom
-    $(map(out_attr(S, c)) do f
```

Comment:
The accessor `out_attr` is not used elsewhere, so either update it or remove it.